config/activator.yaml

```diff
@@ -19,6 +19,7 @@ metadata:
   namespace: knative-serving
   labels:
     serving.knative.dev/release: devel
+    app: deployment
```

Comment:
Just curious, is this change required for line 63 in `test/performance/metrics/runtime.go`?

Reply:
I thought removed this.

Reply:
Stupig git. Pushed now.